ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -289,9 +289,10 @@ function getValues(pendingApproval, t, actions, history, data) {
             [t('blockExplorerUrl')]: t('blockExplorerUrlDefinition'),
           },
           warnings: {
-            [t('networkURL')]: isValidASCIIURL(customRpcUrl)
-              ? undefined
-              : t('networkUrlErrorWarning', [toPunycodeURL(customRpcUrl)]),
+            [t('networkURL')]:
+              !customRpcUrl || isValidASCIIURL(customRpcUrl)
```

Comment:
Nit: `isValidASCIIURL` takes care of scenario when `customRpcUrl` is not defined.

Reply:
we don't want to show the error if there is no URL passed so we need to check !customRpcUrl here